### DISCUSSION TITLE
How to dynamically set invalid property?

### DISCUSSION THREAD

**Discussion by nerisonpitogo** [livewire/flux#885] [Feature Requests]
This is a working code but permanently adds the invalid property. 

```
<div x-data="{ withError: $wire.withError[{{ json_encode($pump->id) }}] }" x-init="alert(withError)">
      <flux:cell>
          <flux:input invalid
              x-on:keyup="updateValues({{ $pump->id }}, '{{ $pump->tank->gas_type }}')"
              type="number" min="1" wire:model="initialReadings.{{ $pump->id }}">
          </flux:input>
      </flux:cell>
  </div>
```

Now I want to dynamically set the invalid based on the withError - which is currently true

```
                        <div x-data="{ withError: $wire.withError[{{ json_encode($pump->id) }}] }" x-init="alert(withError)">
                            <flux:cell>
                                <flux:input x-bind:invalid="withError"
                                    x-on:keyup="updateValues({{ $pump->id }}, '{{ $pump->tank->gas_type }}')"
                                    type="number" min="1" wire:model="initialReadings.{{ $pump->id }}">
                                </flux:input>
                            </flux:cell>
                        </div>
```

This does not add red border. The generated code is this. 

![image](https://github.com/user-attachments/assets/600f1466-fdcb-4bdd-865e-2bf2b72fa181)

And also I tried the following but returns an error.

<flux:cell>
      <flux:input @if ($withError[$pump->id]) invalid @endif
          x-on:keyup="updateValues({{ $pump->id }}, '{{ $pump->tank->gas_type }}')"
          type="number" min="1" wire:model="initialReadings.{{ $pump->id }}">
      </flux:input>
  </flux:cell>
  
  [2024-12-18 17:59:31] local.ERROR: syntax error, unexpected token "endif" (View: D:\flux\cbar\resources\views\livewire\station\calculator\calculator.blade.php) {"userId":1,"exception":"[object] (Illuminate\\View\\ViewException(code: 0): syntax error, unexpected token \"endif\" (View: D:\\flux\\cbar\\resources\\views\\livewire\\station\\calculator\\calculator.blade.php) at D:\\flux\\cbar\\storage\\framework\\views\\cab71d16f57034744163f441154aa18e.php:804)
[stacktrace]

**Comment by joshhanley** [comment]
@nerisonpitogo this doesn't look like a feature request. Please open an issue.
